Dockerfile.builder

```diff
@@ -49,4 +62,9 @@ WORKDIR $WORKSPACE/
 
 VOLUME /dist
 
-ENTRYPOINT ["/bin/bash"]
+COPY docker-entrypoint.sh /home/fabric8/fabric8-planner/
```

Comment:
1. If I understand it correctly, you don't need to copy `docker-entrypoint.sh` because all files are already copied (see line 57)
2. `/home/fabric8/fabric8-planner` should be replaced with `$WORKSPACE`